src/main/java/seedu/address/model/event/EventTime.java

```diff
@@ -11,7 +11,7 @@
 public class EventTime {
     public static final String MESSAGE_CONSTRAINTS = "Invalid date and time entered. Please follow this format: "
             + System.lineSeparator() + "day-month-year 24h time (d-M-uuuu HHmm)";
-    public static final LocalDateTime nullValue = LocalDateTime.parse("2020-02-02T12:00");
+    public static final LocalDateTime NULL_VALUE = LocalDateTime.parse("2020-02-02T12:00");
```

Comment:
Why u change EventTime when ur PR is about finding task?

Reply:
This is because the CI checks will not pass unless I changed this. This is a static field so the CI checks required it to follow the coding standards of a static field.